### PR TITLE
fix(lint): rule noChildrenProp not valid under children in a new line

### PR DESCRIPTION
**Pull request by fireairforce**
closes: https://github.com/biomejs/biome/issues/2211

### CODE REVIEW

**Comment by fireairforce** [comment]
Ci passed at my local: 

image

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Formatter
 
+#### Bug fixes
+
+- Fix [#2211](https://github.com/biomejs/biome/issues/2211). noChildrenProp should work fine when children pass as a prop in a new line. Contributed by @fireairforce
+
```
Comment (Sec-ant) [inline CHANGELOG.md:31]: It should be put in the linter section.

Comment (fireairforce) [reply CHANGELOG.md:31]: ok

Comment (fireairforce) [reply CHANGELOG.md]: done~

**Comment by Sec-ant** [comment]
Thank you!

**Pull request merged**